<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorTable</class>
 <widget class="QWidget" name="BehaviorTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="reverse">
       <property name="text">
        <string>Reverse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirror">
       <property name="text">
        <string>Mirror</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSelection">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="motionCurve">
       <property name="text">
        <string>Motion Curve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="servoValue"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addFrame">
       <property name="text">
        <string>Add All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
